python/utils/scurl.sh

```diff
@@ -42,9 +54,20 @@ for item in "$@" ; do
     fi
     if [ "$item" == "--key" ]; then
         next_is_privk=true
+        if [ -n "$DISABLE_CLIENT_AUTH" ]; then
+            continue
+        fi
+    fi
+    if [ "$item" == "--cert" ]; then
+        next_is_cert=true
+        if [ -n "$DISABLE_CLIENT_AUTH" ]; then
+            continue
+        fi
     fi
+    fwd_args+=("$item")
 done
 
+
 if [ -z "$privk" ]; then
```

Comment:
I think we should add a check like this to ensure a `--cert` was passed, since we now _require_ a cert.